main.tf

```diff
@@ -16,7 +22,7 @@ resource "google_cloudfunctions_function_iam_member" "member" {
   cloud_function = var.cloud_function
   region         = var.region
   role           = var.role
-  member         = each.value
+  member         = try(var.computed_members_map[regex("^computed:(.*)", each.value)[0]], each.value)
```

Comment:
please update the `google_cloudfunctions_function_iam_policy` for computed members as well